src/fragments/lib/in-app-messaging/js/getting-started.mdx

```diff
@@ -0,0 +1,114 @@
+Make sure you have completed the below steps:
+
+- [Completed the pre-requisite steps.](/lib/in-app-messaging/prerequisites)
+- [Create a campaign from the pinpoint console.](/lib/in-app-messaging/create-campaigns)
+
+## Integrate your application
+
+In your application directory, you should first Install necessary dependencies.Although Amplify In-App Messaging can be used as a standalone JavaScript library, the only UI integration currently available is only for _React Native_. The integrations steps here assume a React Native application as the implementation target.
+
+```js
+npm install aws-amplify@in-app-messaging &&
+npm install aws-amplify-react-native@in-app-messaging &&
+npm install amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage @react-native-picker/picker
+```
+
+You will also need to install pod dependencies for `iOS`. Navigate the the ios directory in your project and run the install command.
+
+```bash
+pod install
+```
+
```

Comment:
We should insert a section here about importing the polyfills.
```suggestion
The AWS SDK requires React Native applications to polyfill `crypto.getRandomValues` and `URL`. Import it at the top of your application's root entry point. For example, `index.js`.
```js
import 'react-native-get-random-values';
import 'react-native-url-polyfill/auto';
```

```